storm-core/src/clj/backtype/storm/daemon/worker.clj

```diff
@@ -290,6 +290,7 @@
       :transfer-fn (mk-transfer-fn <>)
       :assignment-versions assignment-versions
       :backpressure (atom false) ;; whether this worker is going slow
+      :transfer-backpressure (atom false) ;; whether this worker is going slow
```

Comment:
:transfer-backpressure may need to have a different comment from :backpressure variable.
For example, whether this worker's transfer queue gets congested.